### DISCUSSION TITLE
Figure alignment different with/without caption

### DISCUSSION THREAD

**Discussion by austin-hoover** [jgm/pandoc#10391] [Q&A]
index.md:
```
Some text.

![](image.png){width=150px}

![This is a figure caption.](image.png){width=150px}
```

Running `pandoc -o index.html index.md` gives

<img width="273" alt="Screenshot 2024-11-16 at 4 24 14 PM" src="https://github.com/user-attachments/assets/37caf10d-31ba-4903-a675-3db6b4e649f0">

Is this expected behavior?

**Comment by jpcirrus** [comment]
Yes. The description in the manual for the pandoc markdown extension `implicit_figures` states:

> An image with nonempty alt text, occurring by itself in a paragraph, will
       be rendered as a figure with a caption.

Your first example is just an image in a paragraph, not a figure. To turn it into a figure, and so centered, you can use a non-printing character, for example a non-breaking space, as a "caption":

```markdown
![&nbsp;](image.png){width=150px}
```
